neo/Network/RPC/RpcServer.cs

```diff
@@ -297,7 +297,21 @@ private JObject Process(string method, JArray _params)
                         return json;
                     }
                 case "getrawmempool":
-                    return new JArray(Blockchain.Singleton.GetMemoryPool().Select(p => (JObject)p.Hash.ToString()));
+                    {
+                        bool shouldGetUnverified = _params.Count >= 1 && _params[0].AsBooleanOrDefault(false);
+                        if (!shouldGetUnverified)
+                            return new JArray(Blockchain.Singleton.GetMemoryPoolVerified().Select(p => (JObject)p.Hash.ToString()));
```

Comment:
@shargon this line Iine returns the same thing for `getrawmempool` as before this change; so you should receivie the same thing for the call before or after this change.

Reply:
Only if the new parameter is present will the response format be different from before; therefore, existing clients will not be broken, and new versions of the client that pass with a parameter of true can know how to handle the new response format. So the parameter passed is already like something to specify the format desired.

Reply:
So there is an option on the setup?
It is an interesting idea, Jeff, because avoid attacks with this type of request.

Reply:
You are right xD, sorry for my mistake